### PR TITLE
Utilize woonerf bootstrapping

### PR DESCRIPTION
**Pull request by trevorgerhardt**

### CODE REVIEW

File: lib/routes.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+/**
+ * Main entry point for the scenario editor
```
Comment (mattwigway) [inline lib/routes.js]: Comment needs updating.

File: lib/utils/auth0.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ import messages from './messages'
```
Comment (mattwigway) [inline lib/utils/auth0.js]: Doesn't/shouldn't woonerf handle a lot of this auth0 boilerplate?

File: lib/utils/authenticated-fetch.js [hunk 3]
```diff
@@ -2,13 +2,19 @@ import dbg from 'debug'
 import fetch from 'isomorphic-fetch'
```
Comment (mattwigway) [inline lib/utils/authenticated-fetch.js]: I thought we were eliminating authenticatedFetch?

File: lib/utils/auth0.js (hunk 2 above)
Comment (trevorgerhardt) [inline lib/utils/auth0.js]: Wanted the changes to be incremental so we're keeping this Auth0 util for now. Will change that in the future.

File: lib/utils/authenticated-fetch.js (hunk 3 above)
Comment (trevorgerhardt) [inline lib/utils/authenticated-fetch.js]: Same as the last comment. Will remove this in the future.

File: lib/routes.js (hunk 1 above)
Comment (trevorgerhardt) [inline lib/routes.js]: Updated 👌

**Pull request merged**